src/components/Dashboard/FilingHistoryList/DetailsList.vue

```diff
@@ -3,12 +3,12 @@
     <div class="title-bar">
       <h4>
         <v-icon small>mdi-message-reply</v-icon>
-        Detail{{filing.comments.length > 1 ? "s" : ""}} ({{filing.comments.length}})
+        <span class="ml-1">Detail{{filing.comments.length > 1 ? "s" : ""}} ({{filing.comments.length}})</span>
```

Comment:
Icons followed by spans get a base class, if they are in a button. This isn't in a button so we have to apply CSS explicitly.